index.bs

```diff
@@ -5694,9 +5689,11 @@ At this time, one [=credential property=] is defined: the [=resident key credent
             (i.e., <dfn dfn-type="dfn">client-side discoverable credential property</dfn>),
             is a Boolean value indicating whether the {{PublicKeyCredential}} returned as a result of a [=registration ceremony=]
             is a [=client-side discoverable credential=].
-            If {{rk}} is [TRUE], the credential is a [=discoverable credential=];
+            If {{rk}} is [TRUE], the credential is a [=discoverable credential=].
             if {{rk}} is [FALSE], the credential is a [=server-side credential=].
             If {{rk}} is not present, it is not known whether the credential is a [=discoverable credential=] or a [=server-side credential=].
+
+            Note: some [=authenticators=] create [=discoverable credentials=] even when not requested by the user agent. Because of this, user agents may be forced to omit the {{rk}} property because they lack the assurance to be able to set it to [FALSE]. [=[RPS]=] should assume that, if the `credProps` extension is supported, then user agents will endeavour to populate the {{rk}} property. Therefore a missing {{rk}} indicates that the user agent believes that the created credential is mostly likely a [=non-discoverable credential=].
```

Comment:
```suggestion
            Note: some [=authenticators=] create [=discoverable credentials=] even when not requested by the [=client platform=]. Because of this, [=client platforms=] may be forced to omit the {{rk}} property because they lack the assurance to be able to set it to [FALSE]. [=[RPS]=] should assume that, if the `credProps` extension is supported, then user agents will endeavour to populate the {{rk}} property. Therefore a missing {{rk}} indicates that the created credential is most likely a [=non-discoverable credential=].
```